### PR TITLE
{bio}[foss/2018a] Tephra v0.12.0, BioPerl v1.7.2

### PR DESCRIPTION
**Pull request by ebirn**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/Tephra/Tephra-0.12.0-foss-2018a-Perl-5.26.1.eb [hunk 1]
```diff
@@ -0,0 +1,120 @@
+
+easyblock = 'Bundle'
+
+name = 'Tephra'
+version = '0.12.0'
+versionsuffix = '-Perl-5.26.1'
+
+homepage = 'https://github.com/sestaton/tephra'
+description = """Dependency Bundle for Tephra package"""
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/t/Tephra/Tephra-0.12.0-foss-2018a-Perl-5.26.1.eb]: Yes please change this to a description of Tephra.

File: easybuild/easyconfigs/t/Tephra/Tephra-0.12.0-foss-2018a-Perl-5.26.1.eb [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+dependencies = {
```
[the remark this answers is not in the document]
Comment (akesandgren) [reply easybuild/easyconfigs/t/Tephra/Tephra-0.12.0-foss-2018a-Perl-5.26.1.eb]: dependencies is a list, i.e. [], not {}

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 5 out of 6 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/9c92db13f04cbb858fbd446590266c59 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/5be1ba6730ea81d897e330be1ca516fd for a full test report.

**Comment by verdurin** [comment]
```
== 2020-10-16 17:36:12,340 extensioneasyblock.py:110 DEBUG Using start_dir: /eb/maint/build/Tephra/0.12.0/foss-2018a-Perl-5.26.1/Tephra/tephra-0.12.0
== 2020-10-16 17:36:12,340 run.py:203 DEBUG run_cmd: running cmd  perl Makefile.PL PREFIX=/eb/maint/software/Tephra/0.12.0-foss-2018a-Perl-5.26.1  (in /eb/maint/build/Tephra/0.12.0/foss-2018a-Perl-5.26.1/Tephra/tephra-0.12.0)
== 2020-10-16 17:36:12,340 run.py:222 INFO running cmd:  perl Makefile.PL PREFIX=/eb/maint/software/Tephra/0.12.0-foss-2018a-Perl-5.26.1
== 2020-10-16 17:36:14,026 build_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): cmd " perl Makefile.PL PREFIX=/eb/maint/software/Tephra/0.12.0-foss-2018a-Perl-5.26.1 " exited with exit code 21 and output:
Generating a Unix-style Makefile
Writing Makefile for Tephra
Writing MYMETA.yml and MYMETA.json
=====> Configuring Tephra, this may take some time...
[ERROR]: move failed: Is a directory
 (at easybuild/tools/run.py:533 in parse_cmd_output)
```

**Comment by boegel** [comment]
@verdurin Probably makes more sense to create a separate PR for the latest version of `Tephra` with a recent toolchain, and close this?

**Comment by boegel** [comment]
I'm closing this as is, clearly needs more effort to get ready.

@ebirn If it's still useful, please open a new pull request. My apologies for not following up on this more closely...

**Pull request closed without merging**